benches/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+#![feature(test)]
+
+extern crate test;
+extern crate wasmi;
+
+use std::error;
+use std::fs::File;
+use wasmi::{ImportsBuilder, Module, ModuleInstance, NopExternals, RuntimeValue};
+
+use test::Bencher;
+
+// Load a module from a file.
+fn load_from_file(filename: &str) -> Result<Module, Box<error::Error>> {
+	use std::io::prelude::*;
+	let mut file = File::open(filename)?;
+	let mut buf = Vec::new();
+	file.read_to_end(&mut buf)?;
+	Ok(Module::from_buffer(buf)?)
+}
+
+#[bench]
+fn bench_tiny_keccak(b: &mut Bencher) {
+	let wasm_kernel = load_from_file(
+		"./wasm-kernel/target/wasm32-unknown-unknown/release/wasm_kernel.wasm",
+	).expect("failed to load wasm_kernel. Is `build.rs` broken?");
```

Comment:
expect could be inside the `load_from_file`

Reply:
True, but in this case it will take a few more expects with less clear and/or redundant messages. Precisely because I wanted to avoid this and use `?` I extracted one expect to the outside of the function.